scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -526,9 +526,10 @@
                                         (constraints/make-fenzo-group-constraint
                                           db group #(guuid->considerable-cotask-ids (:group/uuid group)) running-cotask-cache))
                                       (:group/_job job)))))
```

Comment:
Nitpick: Could have just threaded this in the threading macro.

Reply:
Didn't see any idiomatic way to do so ---- macro uses ->, not ->>, so the list would be in the wrong spot.